.github/workflows/test-ios-device.yaml

```diff
@@ -30,27 +69,100 @@ jobs:
+
+        # RESULTS_DIR_NAME is created by run_android_testlab script
+      - name: Get test outputs from Google Cloud Storage
         if: success() || failure()
+        run: |
+          GSUTIL_PATH="gs://patrol_runs/${{ env.RESULTS_DIR_NAME }}/${{ matrix.device-model }}-${{ matrix.device-version }}-en_US-portrait"
+          mkdir "test_outputs"
+          gsutil -m cp -r "$GSUTIL_PATH/TestLogs/*" "test_outputs"
+          gsutil -m cp -r "$GSUTIL_PATH/*.xml" "test_outputs"
+          gsutil -m cp -r "$GSUTIL_PATH/*.mp4" "test_outputs"
+          gsutil -m cp -r "$GSUTIL_PATH/*.txt" "test_outputs"
 
-      - name: Run notifications_single_test.dart
-        run: ./run_ios_testlab integration_test/notifications_single_test.dart
+      - name: Find xcresult path
         if: success() || failure()
+        run: echo "XCRESULT_PATH=$(find ${{ github.workspace }}/packages/patrol/example/test_outputs -name "*.xcresult" -type d)" >> $GITHUB_ENV
 
-      - name: Run notifications_many_test.dart
-        run: ./run_ios_testlab integration_test/notifications_many_test.dart
+      - name: Publish test report to summary
         if: success() || failure()
+        uses: kishikawakatsumi/xcresulttool@v1
+        with:
+          title: Patrol tests on ${{ matrix.device-model }} iOS ${{ matrix.device-version }}
+          upload-bundles: never
+          path: |
+            ${{ env.XCRESULT_PATH }}
 
-      - name: Run open_app_test.dart
-        run: ./run_ios_testlab integration_test/open_app_test.dart
+      - name: Upload XML test report to artifacts
+        uses: actions/upload-artifact@v3
         if: success() || failure()
+        with:
+          name: XML test report
+          path: ${{ github.workspace }}/packages/patrol/example/test_outputs/*.xml
 
-      - name: Run open_quick_settings_test.dart
-        run: ./run_ios_testlab integration_test/open_quick_settings_test.dart
+      - name: Upload XCRESULT test result to artifacts
+        uses: actions/upload-artifact@v3
         if: success() || failure()
+        with:
+          name: Test-Runner.xcresult
+          path: ${{ github.workspace }}/packages/patrol/example/test_outputs/*.xcresult
+
+      - name: Upload captured video to artifacts
+        uses: actions/upload-artifact@v3
+        if: success() || failure()
+        with:
+          name: Captured video
+          path: ${{ github.workspace }}/packages/patrol/example/test_outputs/*.mp4
+
+      - name: Upload device logs to artifacts
+        if: success() || failure()
+        uses: actions/upload-artifact@v3
+        with:
+          name: Device logs
+          path: ${{ github.workspace }}/packages/patrol/example/test_outputs/*.txt
+
+  call_send_slack_message:
+    name: Notify on Slack
+    uses: ./.github/workflows/send-slack-message.yaml
+    needs: run_tests
+    if: always()
+    with:
+      TESTS_EXIT_CODE: ${{ needs.run_tests.outputs.TESTS_EXIT_CODE }}
+      WORKFLOW_FRIENDLY_NAME: "Filip is :gigachad:"
```

Comment:
I agree wholeheartedly, but now that everybody knows this, let's change this value to something else.